components/create_post/create_post.jsx

```diff
@@ -1195,7 +1202,7 @@ class CreatePost extends React.PureComponent {
         }
 
         let preview = null;
-        if (!readOnlyChannel && (draft.fileInfos.length > 0 || draft.uploadsInProgress.length > 0)) {
+        if (!this.isReadOnlyChannel() && (draft.fileInfos.length > 0 || draft.uploadsInProgress.length > 0)) {
```

Comment:
Nit: we could assign the value at the top of the render function

Reply:
I was wondering about that and if that is our preferred practice.